src/components/Header/Header.tsx

```diff
@@ -75,12 +77,13 @@ const Header = ({
             navbarStatus === true ? styles.showMenu : styles.hideNav
           )}
         >
-          {menuItemData.map((name, index) => {
+          {menuItemData.map((menuItem, index) => {
             return (
               <MenuItem
                 key={`menuItem-${index}`}
                 closeMenu={closeMenu}
-                name={name}
+                name={menuItem.title}
```

Comment:
try to keep naming consistent so either name or title. if you keep it the same you can then do `{...{path, title}}` for clean props.